lib/bcu/command/upgrade.rb

```diff
@@ -88,42 +85,21 @@ def install(app, options)
 
       begin
         # Force to install the latest version.
-        success = system "brew cask install #{options.install_options} #{app[:token]} --force " + verbose_flag
+        cmd = "brew cask reinstall #{options.install_options} #{app[:token]} --force " + verbose_flag
```

Comment:
>Warning: Calling brew cask reinstall is deprecated! Use brew reinstall instead.

This should probably be just

```suggestion
        cmd = "brew reinstall #{options.install_options} #{app[:token]} --force " + verbose_flag
```

(untested)

Reply:
Hey @Amorymeltzer , thx for the comment. I checked and that is still for unreleased version (as `stable` doesn't have that warning yet). But I checked that `brew reinstall [CASK]` works as well and will fix that in [next PR](https://github.com/buo/homebrew-cask-upgrade/pull/173).